instrumentation/src/main/java/io/opentelemetry/android/features/diskbuffering/scheduler/ExportScheduleHandler.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.features.diskbuffering.scheduler
+
+/**
+ * Sets up a scheduling mechanism to read and export previously stored signals in disk.
+ */
+interface ExportScheduleHandler {
```

Comment:
Not for this PR:
If using the Kotlin `explicitApi` mode, all those interfaces have to be made `public`, then you can use the `org.jetbrains.kotlinx.binary-compatibility-validator` plugin to generate a public API file its quite nice.

Reply:
That sounds really nice 👍 I think we should create an issue to address it later.

Reply:
Sure, https://github.com/open-telemetry/opentelemetry-android/issues/217
It's quite simple, the plugin does all the magic, all you need to do is install the plugin and make the commands part of the CI pipeline, so the CI generates the API dump in every CI run if any, the check is already done automatically part of the builtin gradle `check`.